### PR TITLE
Update Berkeley migrators to account for omission of `WorkflowChain` class

### PR DESCRIPTION
**Pull request by eecavanna**
In this branch, I updated the migrators so that they no longer do anything with the `workflow_chain_set` collection.

#### Summary of changes

- [x] Deleted migrator (and its "assets" file) that creates the `workflow_chain_set` collection, populates it, and removes the `was_informed_by` field from documents in the various `WorkflowExecutionActivity` collections (e.g. `metegenome_activity_set`)
- [x] Deleted migrator that moves `gold_analysis_project_identifiers` values from documents (whose `type` is `nmdc:MetadenomaAnnotation`) in `workflow_execution_set` to documents in `workflow_chain_set` collection
- [x] Update migrator that consolidates `PlannedProcess` grandchild collections into its child collections, so it no longer does anything with the `workflow_chain_set` collection (which would not have been created in the first place)
- [x] Updated a `make` target in `project.Makefile` so it no longer references the `workflow_chain_set` collection
- [x] **New:** Implemented a migrator that removes the `part_of` field from all documents that represent instances of the `WorkflowExecution` class or any of its subclasses

### CODE REVIEW

**Review by aclum — COMMENTED** [review]
It it isn't happening elsewhere we need a migrator that removes `part_of` from all WorkflowExecution subclasses if the key exists. Not sure if you want to merge this in before doing that or use this branch.

**Comment by eecavanna** [comment]
Thanks! I'll confirm whether such a migrator exists and, if one doesn't, I'll add one to this branch.

**Comment by brynnz22** [comment]
Hey @eecavanna that `part_of` removal migrator @aclum mentioned doesn't exist. We will need to add it. Let me know if would like me to add it?

**Comment by eecavanna** [comment]
Thanks, @brynnz22. I just implemented one that I think does that. Will you take a look? It also does it to all documents in the `workflow_execution_set` collection, itself, although that may not be necessary.

**Comment by brynnz22** [comment]
@eecavanna That migrator looks good! Except do we need to do it run it on the "child collections"? It looks to me like those children collections were deleted in the migrator you wrote creating the parent collections. So I think you only need to run that migrator on the `workflow_execution_set` because those children collections don't exist anymore after the migrator was run.

File: nmdc_schema/migrators/partials/migrator_from_10_2_0_to_11_0_0/migrator_from_PR104_to_PR195.py
Comment (brynnz22) [inline nmdc_schema/migrators/partials/migrator_from_10_2_0_to_11_0_0/migrator_from_PR104_to_PR195.py:1]: On line 104 of this migrator you delete the children collections. So migrator_from_X_to_PR104 only needs to be ran on the `workflow_execution_set`

**Comment by eecavanna** [comment]
Hey @brynnz22, I designed this one to be run right before the one that consolidates all the child collections into the parent collection. That's why I have it processing each child collection.

`migrator_from_PR104_to_PR195` is the one that consolidates the collections.

I put this one right before that in the `__init__.py` file, which is where the "correct order" is now defined.

**Comment by brynnz22** [comment]
> Hey @brynnz22, I designed this one to be run right before the one that consolidates all the child collections into the parent collection. That's why I have it processing each child collection.
> 
> `migrator_from_PR104_to_PR195` is the one that consolidates the collections.
> 
> I put this one right before that in the `__init__.py` file, which is where the "correct order" is now defined.

Oh okay I see! In that case, then I think the `workflow_execution_set` doesn't yet exist, so should that be removed from the new migrator?

**Comment by eecavanna** [comment]
Oh, I didn't realize `workflow_execution_set` wouldn't exist yet. The adapters do support running `process_each_document` on collections that don't exist (the function basically just `return`s immediately—it doesn't raise an exception or anything).

**Review by brynnz22 — APPROVED** [review]
Okay sounds good.

**Pull request merged**